libraries/resource_providers/api2000/c7000/volume_attachment_provider.rb

```diff
@@ -0,0 +1,20 @@
+# (c) Copyright 2020 Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+module OneviewCookbook
+  module API2000
+    module C7000
+      # VolumeAttachment API2000 C7000 provider
+      class VolumeAttachmentProvider < API1800::C7000::VolumeAttachmentProvider
```

Comment:
Add OneviewCookbook for all library files.
```
      class VolumeAttachmentProvider < OneviewCookbook::API1800::C7000::VolumeAttachmentProvider
```

Reply:
Done